app.py

```diff
@@ -1,55 +1,267 @@
 """
-    Base of app and location where required start queries are performed.
+    README -- Organization of Callback Functions
+
+    In an effort to compartmentalize our development where possible, all callbacks directly relating
+    to pages in our application are in their own files.
+
+    For instance, this file contains the layout logic for the index page of our app-
+    this page serves all other paths by providing the searchbar, page routing faculties,
+    and data storage objects that the other pages in our app use.
+
+    Having laid out the HTML-like organization of this page, we write the callbacks for this page in
+    the neighbor 'app_callbacks.py' file.
 """
+import pstats
+import cProfile
+from db_interface.AugurInterface import AugurInterface
 import dash
-import dash_labs as dl
+from dash import html, dcc
 import dash_bootstrap_components as dbc
-import numpy as np
 from dash_bootstrap_templates import load_figure_template
-import logging
-
-from db_interface.AugurInterface import AugurInterface
-import os
+import numpy as np
 import sys
+import os
+
+import logging
 
 logging.basicConfig(level=logging.DEBUG)
 
-# Get config details
```

Comment:
add "Augur instance configuration details, will be further referenced to as config"